js/ui/handler/scroll_zoom.js

```diff
@@ -146,31 +151,31 @@ ScrollZoomHandler.prototype = {
 
 
 /**
- * Zoom start event. This event is emitted just before the map begins a transition from one
- * zoom level to another, either as a result of user interaction or the use of methods such as `Map#jumpTo`.
+ * Fired just before the map begins a transition from one zoom level to another,
+ * as the result of either user interaction or methods such as [`Map#jumpTo`](#Map#jumpTo).
```

Comment:
Perfect! I think of clarifying all these things as the point of the PR. It got pretty big, as you can see, so it's inevitable that I made typos and missed things like this. Thanks for looking things through in depth!